src/js/components/Drop/stories/Tooltip.js

```diff
@@ -1,40 +1,188 @@
+        onFocus={() => setOver(true)}
+        onBlur={() => setOver(false)}
+        plain
+        onClick={() => console.log(`Clicked me ${index}`)}
+      >
+        <Box pad={{ vertical: 'small' }} align="center">
+          {icon}
+        </Box>
+      </Button>
+      {ref.current && over && (
+        <Drop
+          align={{ left: 'right' }}
+          target={ref.current}
+          plain
+          trapFocus={false}
+        >
+          <Box pad="small" background="pink">
+            <Text color="white">{name}</Text>
+          </Box>
+        </Drop>
+      )}
+    </Box>
+  );
+};
+
+const SidebarDropButton = ({ icon, name }) => {
+  const [open, setOpen] = useState(false);
+  const [over, setOver] = useState(false);
+
+  const ref = useRef();
+
+  return (
+    <Box fill="horizontal">
+      <DropButton
+        ref={ref}
+        onMouseOver={() => setOver(true)}
+        onMouseLeave={() => setOver(false)}
+        onFocus={() => setOver(true)}
+        onBlur={() => setOver(false)}
+        open={open}
+        onOpen={() => setOpen(true)}
+        onClose={() => setOpen(false)}
+        dropAlign={{
+          top: 'top',
+          right: 'right',
+          left: 'right',
+          bottom: 'top',
+        }}
+        dropContent={<DropContent onClose={() => setOpen(false)} />}
+      >
+        <Box>{icon}</Box>
+      </DropButton>
+      {ref.current && over && (
+        <Drop
+          align={{ left: 'right' }}
+          target={ref.current}
+          plain
+          trapFocus={false}
+        >
+          <Box pad="small" background="pink">
+            <Text color="white">{name}</Text>
+          </Box>
+        </Drop>
+      )}
+    </Box>
+  );
+};
+const Sidebar = () => {
+  return (
+    <Box direction="row" fill>
+      <Box
+        direction="column"
+        basis="15%"
+        justify="between"
+        style={{ border: 'blue solid 4px' }}
+        align="center"
+      >
+        <Box>
+          <Nav>
+            {icons.map(({ icon, name }, index) => {
+              return (
+                <SidebarButton
+                  key={name}
+                  icon={icon}
+                  index={index}
+                  name={name}
+                />
+              );
+            })}
+            {menuIcons.map(({ icon, name }, index) => {
+              return (
+                <SidebarDropButton
+                  key={name}
+                  icon={icon}
+                  index={index}
+                  name={name}
+                />
+              );
+            })}
+          </Nav>
+        </Box>
+        <Box>
+          <SidebarButton icon={<Action />} index="Action" name="Action" />
+        </Box>
       </Box>
+    </Box>
+  );
+};
+
+const Tooltips = () => {
+  return (
+    <Grommet theme={grommet} full>
+      <Sidebar />
     </Grommet>
   );
 };
 
-storiesOf('Drop', module).add('Tooltip', () => <TooltipDrop />);
+storiesOf('Drop', module).add('Tooltips', () => <Tooltips />);
```

Comment:
I think we still need a single Tooltip story so the users can find an easy way to come up with tooltips. So separating Tooltips to a separate story would be a good approach here, unless we can cover it with a unit test and then we won't need the story, thoughts?

Reply:
i gotcha, makes sense. ill have a new, simpler example up soon.